guides/hack/40-contributing/01-introduction.md

```diff
@@ -74,12 +74,12 @@ automatically.
 
 If you want to see old versions of the docs, we provide [regular
 Docker images of this
-site](https://hub.docker.com/r/hhvm/user-documentation/tags). This is
+site](https://hub.docker.com/r/meta/hhvm-user-documentation/tags). This is
```

Comment:
Note that https://hub.docker.com/r/meta/hhvm-user-documentation/tags does not yet exist; will appear after first push to Docker Hub